src/lib.ts

```diff
@@ -6,6 +6,7 @@ import { TWO_POW256 } from 'ethereumjs-util'
 
 let mem: WebAssembly.Memory
 let res: Buffer
+res = Buffer.alloc(32)
```

Comment:
This prevents a crash if `eth2_savePostStateRoot` is never called (cli.ts crashes if `res` is undefined).